CONTRIBUTING.md

```diff
@@ -15,10 +15,8 @@ You are welcome to contribute with your pull requests. These steps explain the c
 4. Make sure the tests pass.
```

Comment:
Seems, that there is still mentioning of the CLA. Tis must be removed.

Reply:
Removed